docker-compose.yml

```diff
@@ -28,13 +28,21 @@ x-logging:
   options:
     max-size: 1m
     max-file: "1"
+x-volumes:
+  &default-volumes
+  - .:/app:cached
+  - bundle:/bundle
+  - node_modules:/app/node_modules
+  - packs:/app/public/packs
+  - rails_cache:/app/tmp/cache
 
 services:
   db:
     image: postgres
     logging: *default-logging
     volumes:
       - ./db/init.sql:/docker-entrypoint-initdb.d/10-init.sql
+      - db:/var/lib/postgresql/data
```

Comment:
By adding volumes here and in redis, data will now be persisted after a `docker-compose down`.  The way to get rid of the volumes is by doing `docker-compose down --volume`.  Unfortunately I'm not sure how to hook these up to the current containers, so you might need to do a `docker-compose down` first to get them to work.  Once you do that for sure it saves it.

Reply:
If you wanted to completely start over from nothing.  If we want to add it that's fine, but I think a normal `docker-compose down` is still useful because there were a lot of time I wanted to do that but didn't want to lose all the data in my DB.

Reply:
Alright np, if you get value out of it then let's leave it as-is